server/src/main/java/org/opensearch/node/Node.java

```diff
@@ -1101,6 +1110,8 @@ public Node start() throws NodeValidationException {
         nodeService.getMonitorService().start();
 
         final ClusterService clusterService = injector.getInstance(ClusterService.class);
+        final ActionModule actionModule = injector.getInstance(ActionModule.class);
+        actionModule.registerAction(ExtensionsAction.INSTANCE, TransportExtensionsAction.class);
```

Comment:
Can we have tests for `registerActions`?

Reply:
Thats a good point. 
Sure.